src/layout/Footer.jsx

```diff
@@ -73,28 +73,23 @@ const Footer = ({ title }) => {
           </Link>
         </Grid>
         <Grid item>
-          <Link to="/" className={classes.footerLink}>
+          <Link to="/about" className={classes.footerLink} component={GatsbyLink}>
             About Us
           </Link>
         </Grid>
         <Grid item>
-          <Link to="/" className={classes.footerLink}>
+          <Link to="/join" className={classes.footerLink} component={GatsbyLink}>
             Write For Us
           </Link>
         </Grid>
         <Grid item>
-          <Link to="/" className={classes.footerLink}>
+          <Link to="/join" className={classes.footerLink} component={GatsbyLink}>
```

Comment:
Is it okay for two links right next to each other to lead to the same page?

Reply:
I did that on purpose, since some people might think they mean different things. I debated making it one link called "Participate" or "Write or Work With Us" but none of them felt right